### PR TITLE
feat: Relocated main.go to avoid renaming the binary after installation

### PR DESCRIPTION
**Pull request by almas1992**
## 📑 Description

Closes https://github.com/goravel/goravel/issues/?

@coderabbitai summary

## ✅ Checks

- [ ] Added test cases for my code

### CODE REVIEW

**Comment by almas1992** [comment]
Review Ready

File: README.md [hunk 1]
```diff
@@ -20,16 +20,7 @@ Goravel Installer is a command-line tool that helps you to install the Goravel f
...
-
-# You can rename the executable file
-# Linux / MacOS
-mv "$GOBIN/installer" "$GOBIN/goravel"
-
-# Windows
-move "%GOBIN%\installer.exe" "%GOBIN%\goravel.exe"
-# Windows Powershell
-move "$Env:gopath\bin\installer.exe" "$Env:gopath\bin\goravel.exe"
+go install github.com/goravel/installer/goravel@latest
```
Comment (hwbrzzl) [inline README.md:23]: returns error?

image

Comment (almas1992) [reply README.md:23]: Test `go install github.com/goravel/installer/goravel@master` before you have new release.

Comment (hwbrzzl) [reply README.md:23]: The same:

image

Comment (almas1992) [reply README.md:23]: you have merged?

**Review by hwbrzzl — APPROVED** [review]

**Comment by hwbrzzl** [comment]
Great catch

**Pull request merged**